requirements.txt

```diff
@@ -1 +1 @@
-pifacecommon==4.0.0
\ No newline at end of file
+git+git://github.com/avishayp/pifacecommon
```

Comment:
This should probably go back to pifacecommon or `setup.py` should be updated. I'm not sure which is the preferred method of specifying dependencies these days.